src/test/java/com/google/javascript/clutz/DepgraphTest.java

```diff
@@ -25,6 +25,6 @@ public void testParseFile() throws Exception {
     assertThat(depgraph.getNonroots())
         .containsExactly("javascript/closure/base.js", "javascript/closure/string/string.js")
         .inOrder();
-    assertThat(depgraph.getExterns()).containsExactly("javascript/common/dom.js").inOrder();
+    assertThat(depgraph.getNonrootExterns()).containsExactly("javascript/common/dom.js").inOrder();
```

Comment:
Add assertion on getRootExterns too

Reply:
Add a test for partialInput flag branch too.

Reply:
Done, done.